06_modules_lab/01.py

```diff
@@ -1,14 +1,5 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times.
-
-For example if program was started as:
-    python 01.py 5
-
-It should print:
-    Hello Python
-    Hello Python
-    Hello Python
-    Hello Python
-    Hello Python
-"""
+#!/usr/bin/python
+import sys
+import os
+for i in range(int(sys.argv[1])):
+	print ('Hello Python')
```

Comment:
Note print is usually not written with parenthesis in python2, unless you use the newer (future) print. So you should either use:

```
print 'hello python'
```

or:

```
from __future__ import print_function
print('hello python')
```